modelling/src/main/java/org/axonframework/modelling/saga/repository/jpa/AbstractSagaEntry.java

```diff
@@ -1,105 +0,0 @@
-/*
- * Copyright (c) 2010-2018. Axon Framework
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.axonframework.modelling.saga.repository.jpa;
-
-import org.axonframework.serialization.SerializedObject;
-import org.axonframework.serialization.Serializer;
-
-import javax.persistence.*;
-
-/**
- * Java Persistence Entity allowing sagas to be stored in a relational database.
- * The serialized type of the @{@link SagaEntry} is declared in the concrete implementation of this class.
- *
- * @author Christophe Bouhier
- * @since 3.0.3
- *
- * @param <T> the serialized content-type of the saga.
- */
-@MappedSuperclass
-public abstract class AbstractSagaEntry<T> {
```

Comment:
I don't think we should remove this one, although it's not used anymore now that the Jakarta and Javax entities are merged. Let's deprecate the class describing the predicament instead.

Reply:
Re-added.